changelog.md

```diff
@@ -41,6 +41,7 @@
 - [FIXED] Nested query return correct result when quoteIdentifiers is false. (Postgres) [#6363](https://github.com/sequelize/sequelize/issues/6363)
 - [FIXED] Fixed an issue where changing multiple ENUM columns in PostgreSQL could break. [#6203] (https://github.com/sequelize/sequelize/issues/6203)
 - [FIXED] Add `parent`, `original` and `sql` properties to `UniqueConstraintError`
+- [CHANGED] Converted over to using docker containers for the dbs in Travis-ci [#4727] (https://github.com/sequelize/sequelize/issues/4727)
```

Comment:
Shouldn't this be `[INTERNAL]` or something like that? Not really an API change, isn't it?